xapian-letor/api/featurelist.cc

```diff
@@ -100,11 +103,14 @@ FeatureList::create_feature_vectors(const Xapian::MSet & mset, const Xapian::Que
     for (Xapian::MSetIterator i = mset.begin(); i != mset.end(); ++i) {
 	Xapian::Document doc = i.get_document();
 	std::vector<double> fVals;
```

Comment:
Xapian coding style is not to use camel-case names like `fVals`.  Please use something like `fvals` or `f_vals` instead.